src/Features/Blockcore.Features.NodeHost/UI/Pages/_Host.cshtml

```diff
@@ -13,7 +14,13 @@
     <title>Dashboard</title>
     <base href="~/" />
     <link rel="stylesheet" href="css/bootstrap/darkstrap.min.css" />
-    <link rel="stylesheet" href="css/site.css"/>
+    <link rel="stylesheet" href="css/site.css" />
+    @{
+        if (this.Network.CoinTicker == "RUTA")
```

Comment:
This should be change into some sort of HTTP handler that returns favicon based on query and dynamically loads it, like was done in the old legacy Explorer.

Reply:
yes a general practice is to try to avoid `If (network == XX)` conditions and use inheritance or interfaces otherwise the code will look very boated and hard coded

Reply:
@sondreb do you have a link or additional information about the 'legacy Explorer' .. I am not sure which one is the one you are mentioning